setup/install.sh

```diff
@@ -10,8 +10,8 @@ set -x
 
 # Install foundry
 echo "===== 1. INSTALLING DEPENDENCIES ====="
-sudo dnf install https://rpm.nodesource.com/pub_20.x/nodistro/repo/nodesource-release-nodistro-1.noarch.rpm -y
-sudo dnf install nodejs -y --setopt=nodesource-nodejs.module_hotfixes=1
+curl -fsSL https://rpm.nodesource.com/setup_20.x | bash -
+dnf install -y nodejs
```

Comment:
Just a quick question, this doesn't need `sudo`? I just ask as I remember when I was mucking around that the newer Amazon Linux instances were a bit more restricted in terms of user permissions.

Reply:
Uh, no it worked ok for me, but then the rest of the file is sudo, so theres no harm if you'd like me to amend to be consistent

Reply:
Actually, it's probably a good prompt for me to go through and find which commands _do_ need `sudo` and which don't. Best practise would be to _only_ `sudo` the things that need it and run everything else under the more limited user permissions.

I'll merge this, and make a note to go back over the script for this. Thanks for your PR 😄